service/src/java/org/apache/hive/service/cli/operation/Operation.java

```diff
@@ -452,4 +452,8 @@ public String getQueryTag() {
   public String getQueryId() {
     return queryState.getQueryId();
   }
+
+  public boolean showOperationDrilldownLink() {
```

Comment:
I don't feel that this method should be closely connected to this interface

Reply:
Nice find, perhaps I can move the check to the OperationManager.